app/models/attendance.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+# Attendance of a child during a specific cycle for a child case
+class Attendance < UuidApplicationRecord
+  belongs_to :child_case_cycle
+
+  LENGTHS_OF_CARE = %w[part_day full_day full_plus_part_day full_plus_full_day].freeze
```

Comment:
@csprayregen when someone adds an attendance, are we expecting them to choose what length of care it is, or are we calculating that by the number of hours as laid out in their subsidy rule? 

If we're calculating it, can the user change it, if for some reason our calculation gets it wrong?

Reply:
Since we haven't designed these screens yet, I don't think we have clear answers. My assumption, though, is that we're calculating it for them based on the number of hours, as laid out in their subsidy rule. I'm not sure if we'll let them change it.

Reply:
I'll put a note in the model so these comments are captured.